README.md

```diff
@@ -0,0 +1 @@
+# ChessEngine based on AlphaZero
```

Comment:
Name! what do you think about 'Cattus'?

Reply:
Haha sounds nice, does it have an explanation?

Reply:
I thought of "Deep Purple", because of the historic "Deep Blue" chess engine (and the Deep Purple band)

Reply:
Cattus is 'cat' in latin. I liked deepmind model names: Flamingo, Gato, Chinchilla, all animal names.
Deep purple is also pretty cool

Reply:
Yalla I like it

Reply:
cool